### ISSUE TITLE
Screen readers are unable to review text in draft view.

### ISSUE THREAD

**Issue by rkingett** [trelby/trelby#23]
I am a blind screen reader user, using the free and open source screen reader [NVDA](https://www.nvaccess.org/download/) on Windows. When I try and open up a new screenplay, or view an existing one, in fountain, a text area cannot be focused at all with the screen reader. This also means that I am writing without being able to go back and review output. Is there a way I can turn that pane into an multiline edit field so my screen reader can see text within that edit field?

I am happy to report that the main options/menus are properly labeled, with keyboard shortcuts. I am just unable to use this application because of this one issue. I would love to keep using this app as it appears to be many solutions I was looking for all wrapped into one package.

**Comment by anilgulecha** [comment]
Hi @rkingett ,

Apologies - Trelby so far did not consider accessibility. Technically it draws every character, and does not do it in a way that accessibility technologies will even become aware of it. That's why the menus etc work (those are using the system menus), but the page itself doesn't (those are custom built).

I think wxwidgets does support methods for custom controls to become accessible - and if someone would like to step up and provide PRs towards this, it would be most welcome.
